src/components/calcite-value-list-item/calcite-value-list-item.tsx

```diff
@@ -0,0 +1,91 @@
+import {
+  Component,
+  Element,
+  Event,
+  EventEmitter,
+  Host,
+  Listen,
+  Method,
+  Prop,
+  h
+} from "@stencil/core";
+import { ICON_TYPES } from "../calcite-pick-list/resources";
+
+@Component({
+  tag: "calcite-value-list-item",
+  styleUrl: "./calcite-value-list-item.scss",
+  shadow: true
+})
+export class CalciteValueListItem {
+  // --------------------------------------------------------------------------
+  //
+  //  Properties
+  //
+  // --------------------------------------------------------------------------
+
+  @Prop({ reflect: true }) icon: ICON_TYPES | null = null;
+
+  @Prop() selected = false;
+
+  @Prop({ reflect: true }) textHeading: string;
+
+  @Prop({ reflect: true }) textDescription: string;
+
+  @Prop({ reflect: true }) value: string;
+
+  // --------------------------------------------------------------------------
+  //
+  //  Private Properties
+  //
+  // --------------------------------------------------------------------------
+
+  @Element() el: HTMLCalciteValueListItemElement;
+
+  pickListItem = null;
+
+  // --------------------------------------------------------------------------
+  //
+  //  Events
+  //
+  // --------------------------------------------------------------------------
+
+  @Event() calciteValueListItemSelectedChange: EventEmitter;
+
+  @Listen("calcitePickListItemSelectedChange") calcitePickListItemSelectedChangeHandler(event) {
+    event.detail.item = this.el;
+    this.calciteValueListItemSelectedChange.emit(event.detail);
+  }
+
+  // --------------------------------------------------------------------------
+  //
+  //  Public Methods
+  //
+  // --------------------------------------------------------------------------
+
+  @Method() async toggleSelected(coerce?: boolean, emit = false) {
```

Comment:
Suggestion: if you foresee more params, an option object could help keep the API clean.